docs/pages/versions/unversioned/sdk/live-photo.mdx

```diff
@@ -0,0 +1,117 @@
+---
+title: LivePhoto
+description: A library that allows displaying Live Photos on iOS.
+sourceCodeUrl: 'https://github.com/expo/expo/tree/main/packages/expo-live-photo'
+packageName: 'expo-live-photo'
+platforms: ['ios']
+---
+
+> **info** This module is only available from SDK 52 onwards.
+
+import APISection from '~/components/plugins/APISection';
+import { APIInstallSection } from '~/components/plugins/InstallSection';
+import { SnackInline } from '~/ui/components/Snippet';
+
+## Installation
+
+<APIInstallSection />
+
+## Usage
+
+Here's a simple example of `expo-live-photo` usage combined with `expo-image-picker`.
+
+<SnackInline label='LivePhoto' dependencies={['expo-live-photo', 'expo-image-picker', 'react', 'react-native']}>
+
+```tsx
+import * as ImagePicker from 'expo-image-picker';
+import { LivePhotoAsset, LivePhotoView, LivePhotoViewType } from 'expo-live-photo';
+import { Platform } from 'expo-modules-core';
```

Comment:
```suggestion
import { Platform } from 'expo-modules-core';
```

I think the `Platform` here is meant to be imported from `react-native`?

Reply:
It's available in both places, I assumed we prefer using our implementation 🤔 @tsapeta

Reply:
Good catch Aman. In general, in our own code, we should use `Platform` from `expo-modules-core`, but we shouldn't recommend doing this in user's projects, thus we shouldn't use it in examples.

As a general rule – application's code should never import from `expo-modules-core`, this is an internal package. If we ever want to expose our implementation of the `Platform`, it should be re-exported from the `expo` package (public APIs).

Reply:
To be honest I would simplify this example by removing that import and then just
```diff
- expo-live-photo is not available on {Platform.OS} 😕
+ expo-live-photo is not available on this platform 😕
```